src/core/ext/filters/client_channel/lb_policy/weighted_target/weighted_target.cc

```diff
@@ -769,16 +771,9 @@ class WeightedTargetLbFactory : public LoadBalancingPolicyFactory {
 
 }  // namespace
 
-}  // namespace grpc_core
-
-//
-// Plugin registration
-//
-
-void grpc_lb_policy_weighted_target_init() {
-  grpc_core::LoadBalancingPolicyRegistry::Builder::
-      RegisterLoadBalancingPolicyFactory(
-          absl::make_unique<grpc_core::WeightedTargetLbFactory>());
+void RegisterWeightedAverageLbPolicy(CoreConfiguration::Builder* builder) {
```

Comment:
Please call this `WeightedTarget` instead of `WeightedAverage`, to match the name of the policy.

Reply:
fixed